gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/test/java/io/gravitee/rest/api/portal/rest/spring/ResourceContextConfiguration.java

```diff
@@ -601,4 +605,14 @@ public ApiMetadataQueryService apiMetadataQueryService() {
     public ValidateApiDomainService validateApiDomainService() {
         return mock(ValidateApiDomainService.class);
     }
+
+    @Bean
+    public IntegrationQueryService integrationQueryService() {
```

Comment:
This should not be necessary. We should declare the in memory alternative only in https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/inmemory/spring/InMemoryConfiguration.java